openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -230,6 +252,86 @@ def handle(self, *args, **options):  # pylint: disable=unused-argument
             except Exception:  # pylint: disable=broad-except
                 log.exception(
                     "Error trying to dump course %s to neo4j, rolling back",
-                    six.text_type(course.id)
+                    course_string
                 )
                 transaction.rollback()
+                unsuccessful_courses.append(course_string)
+
+            else:
+                COMMAND_LAST_RUN_CACHE.set(course_key)
+                successful_courses.append(course_string)
+
+        return successful_courses, unsuccessful_courses
+
+
+class Command(BaseCommand):
+    """
+    Command to dump modulestore data to neo4j
+
+    Takes the following named arguments:
+      host: the host of the neo4j server
+      port: the port on the server that accepts https requests
```

Comment:
nit: Perhaps make it a bit clearer that this is the port for the neo4j server specifically. Reading the above line, then this one, got me a bit confused until I read more of the method.